src/app/modules/verification/verification.constants.ts

```diff
@@ -0,0 +1 @@
+export const SMS_VERIFICATION_LIMIT = 10000
```

Comment:
we can put this in `shared/util/verification` to keep all the numbers in one place

Reply:
Thanks! Will update in base branch